lib/compiler/passes/generate-js.js

```diff
@@ -639,7 +640,7 @@ function generateJS(ast, options) {
             break;
 
           case op.WHILE_NOT_ERROR:    // WHILE_NOT_ERROR b
-            compileLoop(stack.top() + " !== peg$FAILED", 0);
+            compileLoop(stack.top() + " !== peg$FAILED");
```

Comment:
This was an actual error, right?  Was it generating bad code or did it have any other consequences?

Reply:
Right. Just an oversight, no bad consequences